ui/imports/shared/controls/TransactionDelegate.qml

```diff
@@ -393,14 +628,14 @@ StatusListItem {
                         case TransactionDelegate.TransactionType.Send:
                         case TransactionDelegate.TransactionType.Sell:
                         case TransactionDelegate.TransactionType.Buy:
-                            return "-" + root.formatCurrencyAmount(root.fiatValue, root.currentCurrency)
+                            return "-" + root.rootStore.formatCurrencyAmount(root.fiatValue, root.currentCurrency)
```

Comment:
Is this a dash `-` or a minus sign `−`? :) Different things

Reply:
Design

![image](https://github.com/status-im/status-desktop/assets/11396062/71598f95-a138-4720-a768-e89af858da15)

Reply:
No, in code you have a dash `-`, you should change that to minus `−`

Reply:
@caybro You are right. QtCreator doesn't know difference, but on VSC I could see difference.

Fixed